mbedtls/src/lib.rs

```diff
@@ -79,6 +82,13 @@ mod mbedtls {
     pub use super::*;
 }
 
+/// You need to call `psa_crypto_init()` before calling any function from the SSL/TLS, X.509 or PK modules.
+/// This function is fine to be called mutiple times while ensure underlying initilization function is only
+/// been called only once.
```

Comment:
Is that so? I thought this was only needed for TLS, where we already call it?

We can't put this kind of requirement on our users. If this is needed, we need to call it ourselves.